File: 코딩테스트 고득점 Kit/해시/의상/의상.py
def solution(clothes):
    answer = 1

    dic = {clothe[1]: [] for clothe in clothes}

    for clothe in clothes:
        dic[clothe[1]].append(clothe[0])

    # print(dic)

    for value in dic.values():
        answer *= (len(value)+1)

    return answer-1

File: 코딩테스트 고득점 Kit/해시/의상/의상.swift
import Foundation

func solution(_ clothes:[[String]]) -> Int {
    var answer: Int = 1 
    var dic:[String: Int] = [:]


    for clothe in clothes {
        if let _ = dic[clothe[1]] {
            dic[clothe[1]]? += 1
        } else {
            dic[clothe[1]] = 1
        }
    }
    
    for (ket, value) in dic {
        answer *= (value+1)
    }
    
    
    return answer-1
}